src/common/components/Header/style.less

```diff
@@ -9,3 +9,48 @@
     line-height: @layout-header-height;
   }
 }
+
+.responsive-menu {
+  width: 100%;
+  padding-top: 0;
+  z-index: 3;
+
+  .ant-popover-content {
+    .ant-popover-arrow {
+      display: none;
```

Comment:
i dont think using popover for this is a good idea. It feels like we are abusing it. we can have just a simple absolute div overlaying everything and let the developer of the real-world app make his own responsive menu according to his real design